src/utils/createChainedFunction.ts

```diff
@@ -0,0 +1,14 @@
+export const createChainedFunction = (...funcs: ((...args: any[]) => void)[]) => {
```

Comment:
Not a fan of global functions like that.. shouldn't you create a class with a static function? Are the output of the functions the input to the next? If so, could you consider using http://underscorejs.org/#compose ?